client/src/components/BlogPostForm.tsx

```diff
@@ -17,9 +17,9 @@ type props = {
 
 export default function BlogPostForm({ entry }: props) {
   const navigate = useNavigate();
-  const [title, setTitle] = useState(entry?.title);
-  const [subtitle, setSubtitle] = useState(entry?.subtitle);
-  const [location, setLocation] = useState(entry?.location);
+  const [title, setTitle] = useState(entry?.title || '');
```

Comment:
Although it makes no difference in this case, I prefer `??` over `||` because it's almost always what I want, so it's good to get in the habit of always using it.